library/data/registry_entry_attributes/build.gradle

```diff
@@ -0,0 +1,15 @@
+plugins {
+	id("qsl.module")
+}
+
+qslModule {
+	moduleName = "registry_entry_attributes"
+	version = "1.0.0"
+	library = "data"
+	coreDependencies([
+			"resource_loader"
+	])
+	interLibraryDependencies([
+			"tags"
```

Comment:
And REA is part of the data lib too, which means it's fine.

Reply:
Yeah, I'm just confused with it using `inter` instead of `intra` here I guess. It's saying the opposite of what it means. That's probably a problem for a different PR though.

Reply:
Oh true, did not notice, weird.